utils/build.js

```diff
@@ -166,7 +166,7 @@ function generateBuildConfigs({formatToPath, baseDir, moduleName,
     buildConfigs.push({
       rollupConfig: {
         entry: entry || path.join(baseDir, 'src', 'index.js'),
-        plugins: [devReplacePlugin, ...basePlugins, babelPlugin],
```

Comment:
Oh, I guess I forgot to ask this in the PR description.

I was going to mention that I removed it because I didn't see any reason to be minifying the dev build. I thought maybe it was an oversight?